### PR TITLE
[Security Content] 8.3 - Add Investigation Guides 2

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Adds Investigation guides to the following rules:

- Exporting Exchange Mailbox via PowerShell
- Suspicious Process from Conhost
- Windows Defender Disabled via Registry Modification
- Disabling Windows Defender Security Settings via PowerShell
- Microsoft Windows Defender Tampering
- Suspicious .NET Reflection via PowerShell
- PowerShell Suspicious Payload Encoded and Compressed
- Suspicious PDF Reader Child Process
- Suspicious PowerShell Engine ImageLoad
- Conhost Spawned By Suspicious Parent Process

Add missing configuration directive to the following rules:

- PowerShell Kerberos Ticket Request

### CODE REVIEW

File: rules/windows/defense_evasion_posh_assembly_load.toml
Comment (nastasha-solomon) [inline rules/windows/defense_evasion_posh_assembly_load.toml:81]:
```suggestion
Steps for implementing the logging policy with Advanced Audit Configuration:
```

File: rules/windows/credential_access_posh_request_ticket.toml
Comment (nastasha-solomon) [reply rules/windows/credential_access_posh_request_ticket.toml:65]: Also, should there be a "the" before "Advanced Audit Configuration"? For example:

```suggestion
Steps for implementing the logging policy with the Advanced Audit Configuration:
```

File: rules/windows/defense_evasion_posh_assembly_load.toml
Comment (nastasha-solomon) [reply rules/windows/defense_evasion_posh_assembly_load.toml:81]: Also, should there be a "the" before "Advanced Audit Configuration"? For example:

```suggestion
Steps for implementing the logging policy with the Advanced Audit Configuration:
```

File: rules/windows/defense_evasion_posh_compressed.toml
Comment (nastasha-solomon) [inline rules/windows/defense_evasion_posh_compressed.toml:23]:
```suggestion
makes it available for use in various environments and creates an attractive way for attackers to execute code.
```

File: rules/windows/defense_evasion_posh_compressed.toml
Comment (nastasha-solomon) [inline rules/windows/defense_evasion_posh_compressed.toml:70]:
```suggestion
identified. Reset passwords for these accounts and other potentially compromised credentials such as email, business
```

File: rules/windows/defense_evasion_posh_compressed.toml
Comment (nastasha-solomon) [inline rules/windows/defense_evasion_posh_compressed.toml:82]:
```suggestion
Steps for implementing the logging policy with Advanced Audit Configuration:
```

Comment (nastasha-solomon) [reply rules/windows/defense_evasion_posh_compressed.toml:82]: Also, should there be a "the" before "Advanced Audit Configuration"? For example:

```suggestion
Steps for implementing the logging policy with the Advanced Audit Configuration:
```

File: rules/windows/defense_evasion_posh_compressed.toml
Comment (nastasha-solomon) [inline rules/windows/defense_evasion_posh_compressed.toml:91]:
```suggestion
Steps for implementing the logging policy via registry:
```

File: rules/windows/execution_suspicious_powershell_imgload.toml
Comment (nastasha-solomon) [inline rules/windows/execution_suspicious_powershell_imgload.toml:76]:
```suggestion
Steps for implementing the logging policy with Advanced Audit Configuration:
```

Comment (nastasha-solomon) [reply rules/windows/execution_suspicious_powershell_imgload.toml:76]: In case "the" is needed before "Advanced Audit Configuration":

```suggestion
Steps for implementing the logging policy with the Advanced Audit Configuration:
```

File: rules/windows/credential_access_posh_request_ticket.toml
Comment (nastasha-solomon) [inline rules/windows/credential_access_posh_request_ticket.toml:74]:
```suggestion
Steps for implementing the logging policy through the registry:
```

File: rules/windows/defense_evasion_posh_assembly_load.toml
Comment (nastasha-solomon) [inline rules/windows/defense_evasion_posh_assembly_load.toml:90]:
```suggestion
Steps for implementing the logging policy through the registry:
```

File: rules/windows/execution_suspicious_powershell_imgload.toml
Comment (nastasha-solomon) [inline rules/windows/execution_suspicious_powershell_imgload.toml:85]:
```suggestion
Steps for implementing the logging policy through the registry:
```

File: rules/windows/defense_evasion_posh_compressed.toml
Comment (nastasha-solomon) [inline rules/windows/defense_evasion_posh_compressed.toml:25]:
```suggestion
Attackers can embed compressed and encoded payloads in scripts that load directly into the memory without touching the
```

File: rules/windows/execution_suspicious_pdf_reader.toml
Comment (nastasha-solomon) [inline rules/windows/execution_suspicious_pdf_reader.toml:21]:
```suggestion
PDF is a common file type used in corporate environments and most machines have software to
```

File: rules/windows/execution_suspicious_pdf_reader.toml
Comment (nastasha-solomon) [inline rules/windows/execution_suspicious_pdf_reader.toml:25]:
```suggestion
This rule looks for commonly abused built-in utilities spawned by PDF reader processes, which are likely ```

File: rules/windows/execution_suspicious_pdf_reader.toml
Comment (nastasha-solomon) [inline rules/windows/execution_suspicious_pdf_reader.toml:33]:
```suggestion
- Retrieve PDF documents received and opened by the user that could cause this behavior. Common locations include,
```

File: rules/windows/execution_suspicious_powershell_imgload.toml
Comment (nastasha-solomon) [inline rules/windows/execution_suspicious_powershell_imgload.toml:24]:
```suggestion
makes it available for use in various environments and creates an attractive way for attackers to execute code.
```

File: rules/windows/execution_suspicious_powershell_imgload.toml
Comment (nastasha-solomon) [inline rules/windows/execution_suspicious_powershell_imgload.toml:27]:
```suggestion
"PowerShell without PowerShell" and works by using the underlying System.Management.Automation namespace. This technique and can also bypass
```

File: rules/windows/execution_suspicious_powershell_imgload.toml
Comment (nastasha-solomon) [inline rules/windows/execution_suspicious_powershell_imgload.toml:45]:
```suggestion
  - Use the PowerShell `Get-FileHash` cmdlet to get the files' SHA-256 hash values.
```

File: rules/windows/execution_via_hidden_shell_conhost.toml
Comment (nastasha-solomon) [inline rules/windows/execution_via_hidden_shell_conhost.toml:43]:
```suggestion
  - Use the PowerShell `Get-FileHash` cmdlet to get the files' SHA-256 hash values.
```

File: rules/windows/defense_evasion_posh_assembly_load.toml
Comment (nastasha-solomon) [inline rules/windows/defense_evasion_posh_assembly_load.toml:43]:
```suggestion
  - Use the PowerShell `Get-FileHash` cmdlet to get the files' SHA-256 hash values.
```

File: rules/windows/execution_suspicious_pdf_reader.toml
Comment (nastasha-solomon) [inline rules/windows/execution_suspicious_pdf_reader.toml:42]:
```suggestion
  - Use the PowerShell `Get-FileHash` cmdlet to get the files' SHA-256 hash values.
```

File: rules/windows/execution_via_hidden_shell_conhost.toml
Comment (nastasha-solomon) [inline rules/windows/execution_via_hidden_shell_conhost.toml:21]:
```suggestion
The Windows Console Host, or `conhost.exe`, is the server application for Windows Console APIs and
```

File: rules/windows/execution_via_hidden_shell_conhost.toml
Comment (nastasha-solomon) [inline rules/windows/execution_via_hidden_shell_conhost.toml:25]:
```suggestion
`PowerShell.exe` to bypass application whitelisting and security features. Attackers usually inject these implementations into
```

Comment (nastasha-solomon) [reply rules/windows/execution_via_hidden_shell_conhost.toml:25]: Suggesting this edit to break up the sentence for better readability.

File: rules/windows/execution_via_hidden_shell_conhost.toml
Comment (nastasha-solomon) [inline rules/windows/execution_via_hidden_shell_conhost.toml:32]:
```suggestion
- Investigate abnormal behaviors observed by the subject process such as network connections, registry or file
```

File: rules/windows/execution_suspicious_powershell_imgload.toml
Comment (nastasha-solomon) [inline rules/windows/execution_suspicious_powershell_imgload.toml:65]:
```suggestion
identified. Reset passwords for these accounts and other potentially compromised credentials such as email, business
```

File: rules/windows/execution_via_hidden_shell_conhost.toml
Comment (nastasha-solomon) [inline rules/windows/execution_via_hidden_shell_conhost.toml:67]:
```suggestion
identified. Reset passwords for these accounts and other potentially compromised credentials such as email, business
```

File: rules/windows/execution_suspicious_powershell_imgload.toml
Comment (nastasha-solomon) [inline rules/windows/execution_suspicious_powershell_imgload.toml:26]:
```suggestion
Attackers can use PowerShell without having to execute `PowerShell.exe` directly. This technique is often called
```

**Review by nastasha-solomon — COMMENTED** [review]
Hey, @w0rk3r ! This might look like a lot of edits, but they're all pretty minor. I also flagged a few comment for your review so please take a look at them when you have a moment. Thanks for all your hard work!

**Review by imays11 — APPROVED** [review]

**Review by terrancedejesus — APPROVED** [review]
Approving so it is ready to merge when needed.

File: rules/windows/collection_email_powershell_exchange_mailbox.toml
Comment (w0rk3r) [reply rules/windows/collection_email_powershell_exchange_mailbox.toml:22]: The request is handled by the Exchange Server after command is executed, I'm using the description provided by the official MS Docs there https://docs.microsoft.com/en-us/powershell/module/exchange/new-mailboxexportrequest

File: rules/windows/collection_email_powershell_exchange_mailbox.toml
Comment (w0rk3r) [reply rules/windows/collection_email_powershell_exchange_mailbox.toml:23]: image
https://docs.microsoft.com/en-us/powershell/module/exchange/new-mailboxexportrequest

File: rules/windows/defense_evasion_code_injection_conhost.toml
Comment (w0rk3r) [reply rules/windows/defense_evasion_code_injection_conhost.toml:18]: Using the official description here https://docs.microsoft.com/en-us/windows/console/definitions

File: rules/windows/credential_access_posh_request_ticket.toml
Comment (w0rk3r) [reply rules/windows/credential_access_posh_request_ticket.toml:65]: I'll leave this one out by now, and apply a fix accross all rules that use this verbiage in a separated PR

File: rules/windows/credential_access_posh_request_ticket.toml
Comment (w0rk3r) [reply rules/windows/credential_access_posh_request_ticket.toml:74]: I'll leave this one out by now, and apply a fix accross all rules that use this verbiage in a separated PR

File: rules/windows/collection_email_powershell_exchange_mailbox.toml
Comment (nastasha-solomon) [reply rules/windows/collection_email_powershell_exchange_mailbox.toml:22]: Ah, probably best to keep the content aligned with the official MS docs then. Thanks for clarifying @w0rk3r !

File: rules/windows/defense_evasion_posh_compressed.toml
Comment (w0rk3r) [reply rules/windows/defense_evasion_posh_compressed.toml:25]: This one changes the meaning

File: rules/windows/collection_email_powershell_exchange_mailbox.toml
Comment (nastasha-solomon) [reply rules/windows/collection_email_powershell_exchange_mailbox.toml:23]: Gotcha - feel free to ignore my comment :)

File: rules/windows/defense_evasion_code_injection_conhost.toml
Comment (nastasha-solomon) [reply rules/windows/defense_evasion_code_injection_conhost.toml]: Gotcha, feel free to ignore my edit! :)

**Comment by w0rk3r** [comment]
Hey @nastasha-solomon, thanks for the review, can I get another one?

File: rules/windows/defense_evasion_microsoft_defender_tampering.toml
Comment (joepeeples) [inline rules/windows/defense_evasion_microsoft_defender_tampering.toml:41]: Missing comma:
```suggestion
- This mechanism can be used legitimately. Analysts can dismiss the alert if the administrator is aware of the activity,
```

File: rules/windows/defense_evasion_posh_assembly_load.toml
Comment (joepeeples) [inline rules/windows/defense_evasion_posh_assembly_load.toml:24]:
```suggestion
Attackers can use .NET reflection to load PEs and DLLs in memory. These payloads are commonly embedded in the script,
```

File: rules/windows/execution_suspicious_pdf_reader.toml
Comment (joepeeples) [inline rules/windows/execution_suspicious_pdf_reader.toml:21]: "Deal with" is a little informal: maybe "handle"? Or "open", "access", "read"?
```suggestion
PDF is a common file type used in corporate environments and most machines have software to handle these files. This
```

File: rules/windows/execution_suspicious_pdf_reader.toml
Comment (joepeeples) [inline rules/windows/execution_suspicious_pdf_reader.toml:33]: Missing comma:
```suggestion
are not limited to, the Downloads and Document folders and the folder configured at the email client.
```

File: rules/windows/execution_suspicious_powershell_imgload.toml
Comment (joepeeples) [inline rules/windows/execution_suspicious_powershell_imgload.toml:28]: This sentence is a bit of a run-on (several clauses all joined by just "and"). @nastasha-solomon's suggestion above would help, or here's another possibility. This also changes "whitelisting" to "allowlisting," as long as that doesn't change technical meaning here:
```suggestion
Attackers can use PowerShell without having to execute `PowerShell.exe` directly. This technique, often called
"PowerShell without PowerShell," works by using the underlying System.Management.Automation namespace and can bypass
application allowlisting and PowerShell security features.
```

File: rules/windows/execution_via_hidden_shell_conhost.toml
Comment (joepeeples) [inline rules/windows/execution_via_hidden_shell_conhost.toml:26]: I agree with @nastasha-solomon's comment here; the sentence is long and would benefit by breaking up or restructuring somehow. This is a slight modification of her suggestion:
```suggestion
Attackers often rely on custom shell implementations to avoid using built-in command interpreters like `cmd.exe` and 
`PowerShell.exe` and bypass application allowlisting and security features. Attackers commonly inject these implementations into
legitimate system processes.
```

**Comment by w0rk3r** [comment]
Hey @nastasha-solomon & @joepeeples, can I get a final review on this one to merge?

**Review by terrancedejesus — APPROVED** [review]
LGTM.

**Review by joepeeples — APPROVED** [review]
LGTM!

**Pull request merged**
